backend/src/apiserver/model/run.go

```diff
@@ -363,11 +363,11 @@ var runAPIToModelFieldMap = map[string]string{
 	"storage_state":    "StorageState",
 	"status":           "Conditions",
 	"namespace":        "Namespace",               // v2beta1 API
-	"experiment_id":    "ExperimentId",            // v2beta1 API
+	"experiment_id":    "ExperimentUUID",          // v2beta1 API
```

Comment:
nit: If we can have a simple unit test to catch this in the future then it will be great. Even simply checking whether the key is in the db run_columns could help avoid a lot of the unexpected failure. 
https://github.com/kubeflow/pipelines/blob/b9e08ded48f7dae69f4936660fbdf3dc0ba4bcb4/backend/src/apiserver/storage/run_store.go#L31

Reply:
+1, @HumairAK do you mind adding some unit tests in the same PR?

Reply:
absolutely, will follow up with something

Reply:
@chensun / @Tomcli I've added some unit tests for experiment/job/run/task

pipeline/pipeline_version seems to be handled slightly different so I left that one out, also noticed task fields were off exposed by the tests, so updated them as well.